evalml/automl/automl_search.py

```diff
@@ -1006,6 +1010,14 @@ def search(self, show_iteration_plot=True):
                 f"Best pipeline {self.objective.name}: {best_pipeline['validation_score']:3f}"
             )
         self._searched = True
+        if self.search_iteration_plot is not None:
+            if self.verbose and not interactive_plot:
+                self.search_iteration_plot = self.plot.search_iteration_plot(
+                    interactive_plot=interactive_plot
+                )
+                if pio.renderers.default != "browser":
```

Comment:
This stops the automl tests from mass opening graphs in the browser, which was causing the tests to be really slow.